tests/test_pe_reports.py

```diff
@@ -65,6 +65,7 @@ def test_log_levels(level):
             "2021-01-01",
             "input/",
             "output/",
+            "--db-creds-file=/secrests/database_creds.yml",
```

Comment:
Is this a typo?  Did you mean `--db-creds-file=/secrets/database_creds.yml`?

More importantly, why are you passing in a `--db-creds-file` at all if it is [an optional parameter](https://github.com/cisagov/pe-reports/blob/3d0ef88e9f8376e173731191a6861096924cd133/src/pe_reports/report_generator.py#L11-L13) and it isn't relevant to this test case?

Reply:
@dav3r the `--db-creds-file` is a requirement to get actual report output, and if not present the default arg- `/secrets/database_creds.yml` loads - hence optional. 

If a `database_creds.yml` file is not present the code handles the error in an exception and the module will run to completion. So I see what you mean... if it is not necessary to pass the test, so why add it. Originally, I thought it was good to be included due to continuity of other test args that were included. It will be removed.

Reply:
Resolved in b980e9e

Reply:
Thanks @schmelz-ctr 👍